confirm.js

```diff
@@ -53,6 +53,8 @@ module.exports = function (common, deps) {
        .put(common.makeAPIUrl('/confirm/accept/signup/'+signupId))
        .end(function (err, res) {
         if (err != null) {
+          // TODO: update version of lodash so we can use _.get
```

Comment:
I was surprised too, doubly so when the code bailed out saying that `_.get` was not defined. I wasn't sure how big of a task upgrading to `3.10.1` was going to be (ideally there wouldn't be any breaking API changes according to semver, but I found a couple places where function signatures changed while trying to figure out where `_.get` was, so I didn't have a ton of confidence in just bumping it up). There maybe should be a card for it for `platform-client`.